build.gradle

```diff
@@ -89,15 +89,28 @@ idea {
     }
 }
 
+project.version = '0.7.0'
 ext.versionSource = versionSource
-ext.globalVersion = "0.7.0"
+ext.globalVersion = project.version
 
 // To set fishlibVersion, go to project-local gradle.properties file
```

Comment:
suggestion, while you're in there, kill this comment block,  which doesnt make sense in dhcc